go.mod

```diff
@@ -49,6 +50,12 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e // indirect
 	github.com/cloudflare/circl v1.1.0 // indirect
+	github.com/cockroachdb/errors v1.8.1 // indirect
```

Comment:
Is cockroachdb needed?

Reply:
Same here:
```shell
github.com/cockroachdb/errors
This module is necessary because [github.com/cockroachdb/errors](vscode-file://vscode-app/snap/code/143/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) is imported in:

- github.com/onflow/flow-cli/internal/accounts
-- github.com/onflow/flow-cli/flowkit/gateway
--- github.com/onflow/flow-go/model/flow
---- github.com/onflow/flow-go/model/flow.test
----- github.com/onflow/flow-go/utils/unittest
------ github.com/cockroachdb/pebble
```